itests/manual_onboarding_test.go

```diff
@@ -103,72 +95,24 @@ func TestManualSectorOnboarding(t *testing.T) {
 			var cSectorNum abi.SectorNumber
 			var cRespCh chan kit.WindowPostResp
 
-			if withMockProofs {
-				cSectorNum, cRespCh, _ = minerC.OnboardSectorWithPiecesAndMockProofs(ctx, kit.TestSpt)
-			} else {
-				cSectorNum, cRespCh, _ = minerC.OnboardSectorWithPiecesAndRealProofs(ctx, kit.TestSpt)
-			}
+			cSectorNum, cRespCh, _ = minerC.OnboardSectorWithPieces(ctx, kit.TestSpt)
 			// Miner C should still not have power as power can only be gained after sector is activated i.e. the first WindowPost is submitted for it
 			minerC.AssertNoPower(ctx)
 			// Ensure that the block miner checks for and waits for posts during the appropriate proving window from our new miner with a sector
 			blockMiner.WatchMinerForPost(minerC.ActorAddr)
 
 			// Wait till both miners' sectors have had their first post and are activated and check that this is reflected in miner power
-			waitTillActivatedAndAssertPower(ctx, t, minerB, bRespCh, bSectorNum, uint64(defaultSectorSize), withMockProofs)
-			waitTillActivatedAndAssertPower(ctx, t, minerC, cRespCh, cSectorNum, uint64(defaultSectorSize), withMockProofs)
+			minerB.WaitTillActivatedAndAssertPower(ctx, bRespCh, bSectorNum)
+			minerC.WaitTillActivatedAndAssertPower(ctx, cRespCh, cSectorNum)
 
 			// Miner B has activated the CC sector -> upgrade it with snapdeals
 			// Note: We can't activate a sector with mock proofs as the WdPost is successfully disputed and so no point
 			// in snapping it as snapping is only for activated sectors
 			if !withMockProofs {
-				minerB.SnapDealWithRealProofs(ctx, kit.TestSpt, bSectorNum)
+				minerB.SnapDeal(ctx, kit.TestSpt, bSectorNum)
 				// cancel the WdPost for the CC sector as the corresponding CommR is no longer valid
 				bWdPostCancelF()
 			}
 		})
 	}
 }
-
-func waitTillActivatedAndAssertPower(ctx context.Context, t *testing.T, miner *kit.TestUnmanagedMiner, respCh chan kit.WindowPostResp, sector abi.SectorNumber,
-	sectorSize uint64, withMockProofs bool) {
-	req := require.New(t)
-	// wait till sector is activated
-	select {
-	case resp := <-respCh:
-		req.NoError(resp.Error)
-		req.True(resp.Posted)
-	case <-ctx.Done():
-		t.Fatal("timed out waiting for sector activation")
-	}
-
-	// Fetch on-chain sector properties
-	head, err := miner.FullNode.ChainHead(ctx)
-	req.NoError(err)
-
-	soi, err := miner.FullNode.StateSectorGetInfo(ctx, miner.ActorAddr, sector, head.Key())
-	req.NoError(err)
-	t.Logf("Miner %s SectorOnChainInfo %d: %+v", miner.ActorAddr.String(), sector, soi)
-
-	_ = miner.FullNode.WaitTillChain(ctx, kit.HeightAtLeast(head.Height()+5))
-
-	t.Log("Checking power after PoSt ...")
-
-	// Miner B should now have power
```

Comment:
nit: this comment should change to "Miner should have no power"

Reply:
this is removed from here, the code is moved to https://github.com/filecoin-project/lotus/pull/12092/files#diff-607cce01302fd6d23e59fb8699690252d7a55f88256c11a3112e9995fc591889R1015 and the same comment is there, but it's testing `sectorSize` power, so it _should_ have power, right?